src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/Formatting/CSharpFormattingOptions2.cs

```diff
@@ -419,7 +417,11 @@ static CSharpFormattingOptions()
         }
     }
 
+#if CODE_STYLE
+    internal enum LabelPositionOptions
+#else
     public enum LabelPositionOptions
```

Comment:
I did not clone public enum types such as these into shared layer, as that is too much pain, especially to ensure the fields and values are always in sync. I think we should be fine for enums with this approach.